src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int GROW_FACTOR = 2;
+    private int size;
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private int capacity = DEFAULT_CAPACITY;
+    private int threshold = (int) (capacity * DEFAULT_LOAD_FACTOR);
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resizeMap();
+        }
+        int hash = hashCalculator(key);
```

Comment:
```suggestion
```

Reply:
i need hash while hashmap original has this field
how i can resize map wothout hash?